src/praisonai/praisonai/integrations/compute/tenki.py

```
@@ -0,0 +1,312 @@
+
+    async def shutdown(self, instance_id: str) -> None:
+        import asyncio
+        loop = asyncio.get_running_loop()
+        await loop.run_in_executor(None, self._shutdown_sync, instance_id)
+
+    def _shutdown_sync(self, instance_id: str) -> None:
+        info = self._sandboxes.pop(instance_id, None)
+        if info:
+            try:
+                info["sandbox"].terminate()
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)